test/helpers/PlayerInteractionWrapper.js

```diff
@@ -407,7 +407,11 @@ class PlayerInteractionWrapper {
     }
 
     findCardByName(name, zones = 'any', side) {
-        return this.filterCardsByName(name, zones, side)[0];
+        var cards = this.filterCardsByName(name, zones, side);
+        if (cards.length === 0) {
```

Comment:
Can we check that the count is exactly 1 and throw an error if it's either higher or lower (the error should clearly specify the number). If that ends up breaking a lot of tests you can revert and lmk and I'll do it in a separate PR.

Reply:
@AMMayberry1 I started down that path, checking that anything above 1 throws and error. It broke every test. I can dig in further if you would like.

Reply:
Ah interesting, probably one of the setup methods is using this when it shouldn't be. That's probably out of scope for this PR, can you just add a TODO here marking it to look into later?

Reply:
The TODO is added and everything should be good to go.